packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -499,21 +511,48 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
         }
     }
 
-    private onAxisClick(event: _ModuleSupport.PointerInteractionEvent<'click'>, region: _ModuleSupport.RegionName) {
+    private onAxisButtonClick(coords?: Coords, region?: _ModuleSupport.RegionName) {
         this.onCancel();
-        this.onAxisClickAdding(event, region);
+        this.onAxisClickAdding(coords, region);
     }
 
-    private onAxisClickAdding(
-        event: _ModuleSupport.PointerInteractionEvent<'click'>,
-        region: _ModuleSupport.RegionName
-    ) {
+    private onAxisClickAdding(coords?: Coords, region?: _ModuleSupport.RegionName) {
```

Comment:
Tbh, may as well just have the one function here `onAxisButtonClick()`, rather than calling out to `onAxisClickAdding()`. If/when we do other stuff we can break it up as appropriate.